src/reporting/clients/IExtractionClient.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import type { ExtractionLog, ExtractionRun, ExtractionStatus } from "../interfaces/ExtractionProcess";
+import type { EntityListIterator } from "../iterators/EntityListIterator";
+
+export interface IExtractionClient {
+  /**
+   * Gets Logs of an Extraction Run. This method returns the full list of extraction logs.
+   * @param {string} jobId Unique Identifier of the Extraction Run.
+   * @param {string} accessToken OAuth access token with scope `insights:read`.
+   * @param {number} top the number of entities to load per page.
```

Comment:
`the` -> `The`.

Reply:
capitalized all the first letter in such cases